pr_agent/algo/utils.py

```diff
@@ -379,9 +379,15 @@ def set_custom_labels(variables, git_provider=None):
 
     # Set custom labels
     variables["custom_labels_class"] = "class Label(str, Enum):"
+    counter = 0
+    labels_minimal_to_labels_dict = {}
     for k, v in labels.items():
-        description = v['description'].strip('\n').replace('\n', '\\n')
-        variables["custom_labels_class"] += f"\n    {k.lower().replace(' ', '_')} = '{k}' # {description}"
+        description = "'" + v['description'].strip('\n').replace('\n', '\\n') + "'"
+        # variables["custom_labels_class"] += f"\n    {k.lower().replace(' ', '_')} = '{k}' # {description}"
+        variables["custom_labels_class"] += f"\n    {k.lower().replace(' ', '_')} = {description}"
+        labels_minimal_to_labels_dict[k.lower().replace(' ', '_')] = k
+        counter += 1
```

Comment:
**Suggestion:** Use a dictionary comprehension to create `labels_minimal_to_labels_dict` instead of using a for loop.
```suggestion
    labels_minimal_to_labels_dict = {k.lower().replace(' ', '_'): k for k in labels.keys()}
    for k, v in labels.items():
        description = "'" + v['description'].strip('\n').replace('\n', '\\n') + "'"
        variables["custom_labels_class"] += f"\n    {k.lower().replace(' ', '_')} = {description}"
    variables["labels_minimal_to_labels_dict"] = labels_minimal_to_labels_dict
```